backend/package.json

```diff
@@ -7,8 +7,11 @@
   "license": "UNLICENSED",
   "scripts": {
     "build": "nest build",
+    "build:coverage": "shx rm -rf coverage-e2e/ && nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
+    "start:built": "node ./dist/main.js",
+    "start:built:coverage": "c8 --report-dir=coverage-e2e node ./dist/main.js",
```

Comment:
can you document how `coverage-e2e` and `coverage-nest-e2e` are used and how they're different from one another ? 
I've seen both used in config files.

Reply:
I'll need to talk to you to find a good solution regarding this ^^